data/outreachy.yml

```diff
@@ -0,0 +1,138 @@
+rounds:
+  - name: Summer 2019
+    projects:
+      - title: "Test the OCaml compiler with code coverage tools"
+        description: ""
+        mentee: "Oxana Kostikova"
+        blog: "https://lereenadem.wixsite.com/personal"
+        mentors:
+         - Sébastien Hinderer
+         - Florian Angeletti
+      - title: "Test the OCaml compiler with random tests and a reference interpreter"
+        description: >
+          The aim of this project is to extend an existing testcase-generator for the OCaml compiler,
+          using a reference interpreter (existing or newly developed) to find a lot of bugs in the compiler,
+          and fix as much of them as possible.
+        mentee: "Ulugbek Abdullaev"
+        blog: "https://ulugbekna.github.io"
+        mentors:
+         - Gabriel Scherer
```

Comment:
From what I understood, @jmid was also a mentor of that project, but I might be mixing things up. @jmid , dod you co-mentor @ulugbekna on that project?

Reply:
Yep, I was unofficial co-mentor with @gasche at the time :sweat_smile: 
I'm happy to be listed - but I also fully understand if I'm not - it was unofficial after all :shrug:

Reply:
Oh yes, you could/should be listed here.

Reply:
Added 👍